lib/models/Review.js

```diff
@@ -0,0 +1,30 @@
+const pool = require('../utils/pool');
+
+module.exports = class Review {
+  id;
+  userID;
+  detail;
+
+  constructor(row) {
+    this.id = row.id;
+    this.userID = row.user_id;
+    this.detail = row.detail;
+  }
+
+  static async getByID(id) {
+    const { rows } = await pool.query('SELECT * FROM reviews WHERE id = $1', [
+      id,
+    ]);
+    if (!rows) return null;
```

Comment:
Empty arrays `[]` have a truthy value, so this will never return null. To get around this you can check the arrays "first" element which will be falsy since it doesn't exist.

```js
if (!rows[0]) return null;